client/src/pages/layout/Navbar.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar as NavbarB, NavDropdown, Nav } from "react-bootstrap";
```

Comment:
```suggestion
import { Navbar as NavbarBootstrap, NavDropdown, Nav } from "react-bootstrap";
```

lets follow Frank's advice from your last PR to give components really descriptive names - makes it more easy for future developers to read and understand your code

Reply:
Yeah. I have just remembered after pushed it. Frank made a same feedback. Thanks